tests/python/openassetio/test_log.py

```diff
@@ -114,17 +114,27 @@ def test_when_shouldColorOutput_set_false_then_messages_are_not_colored(self, ca
 class Test_LoggerInterface:
     def test_severity_names(self):
         assert (
-            lg.LoggerInterface.kSeverityNames[lg.LoggerInterface.Severity.kCritical] == "critical"
+            lg.LoggerInterface.kSeverityNames[int(lg.LoggerInterface.Severity.kCritical)]
+            == "critical"
         )
-        assert lg.LoggerInterface.kSeverityNames[lg.LoggerInterface.Severity.kError] == "error"
-        assert lg.LoggerInterface.kSeverityNames[lg.LoggerInterface.Severity.kWarning] == "warning"
         assert (
-            lg.LoggerInterface.kSeverityNames[lg.LoggerInterface.Severity.kProgress] == "progress"
+            lg.LoggerInterface.kSeverityNames[int(lg.LoggerInterface.Severity.kError)] == "error"
         )
-        assert lg.LoggerInterface.kSeverityNames[lg.LoggerInterface.Severity.kInfo] == "info"
-        assert lg.LoggerInterface.kSeverityNames[lg.LoggerInterface.Severity.kDebug] == "debug"
         assert (
-            lg.LoggerInterface.kSeverityNames[lg.LoggerInterface.Severity.kDebugApi] == "debugApi"
+            lg.LoggerInterface.kSeverityNames[int(lg.LoggerInterface.Severity.kWarning)]
+            == "warning"
+        )
+        assert (
+            lg.LoggerInterface.kSeverityNames[int(lg.LoggerInterface.Severity.kProgress)]
+            == "progress"
+        )
+        assert lg.LoggerInterface.kSeverityNames[int(lg.LoggerInterface.Severity.kInfo)] == "info"
+        assert (
+            lg.LoggerInterface.kSeverityNames[int(lg.LoggerInterface.Severity.kDebug)] == "debug"
+        )
+        assert (
+            lg.LoggerInterface.kSeverityNames[int(lg.LoggerInterface.Severity.kDebugApi)]
+            == "debugApi"
         )
```

Comment:
[Consider][Minor] I suspect the inconsistent indentation here is a consequence of consistent line wrapping. Due to how some of these statements split into 4 lines now while some still only take one, maybe consider slightly breaking the rule here for a consistent, (and i think more readable) block.

Reply:
Yeah, it does my nut, but the issue with manually massaging it is that we can't the use auto formatters, and the CI black check will fail as it'll want to reformat it for us...